staker-rewards/lib/frauds.js

```diff
@@ -0,0 +1,81 @@
+import clc from "cli-color"
```

Comment:
What do you think about creating a new class which will contain those functions as we do with `SLACalculator`? We can name it `FraudDetector` or somehow and then we can initialize all the contracts in their `initialize` function to avoid doing this in `isOperatorKeepECDSAFraudulent` and `isKeepECDSATransaction` functions. This way we can also pass the `context` only once, during initialization, which seems more elegant, especially from the unit tests perspective.